protonvpn_cli/cli.py

```diff
@@ -326,20 +326,35 @@ def configure_cli():
 
         user_choice = user_choice.lower().strip()
         if user_choice == "1":
-            set_username_password(write=True)
+            try:
+                set_username_password()
+                time.sleep(1)
+            except KeyboardInterrupt:
+                print()
         elif user_choice == "2":
-            set_protonvpn_tier(write=True)
+            set_protonvpn_tier()
+            time.sleep(1)
+            print()
```

Comment:
I've added the sleep and print on all options to so it doesn't instantly print the menu again and making the user feel like nothing happened.